src/main/kotlin/blackjack/domain/player/Player.kt

```diff
@@ -0,0 +1,31 @@
+package blackjack.domain.player
+
+import blackjack.domain.card.Card
+import blackjack.domain.card.CardDenomination
+
+data class Player(
+    val name: String,
+    val cards: List<Card> = emptyList()
+) {
+    fun getCardSum(): Int {
```

Comment:
이 부분이 질문 주신 부분인 것 같은데요!!
[fold](https://blog.leocat.kr/notes/2020/03/09/kotlin-reduce-and-fold)를 사용하면 아래와 같이 구현할 수 있습니다.

```suggestion
   fun getCardSum() = cards
        .sortedByDescending { card -> card.denomination.order }
        .fold(0) { sum, card ->
            val denomination = card.denomination
            sum + if (denomination == CardDenomination.ACE) {
                if (sum + denomination.value[1] > 21) {
                    denomination.value[0]
                } else {
                    denomination.value[1]
                }
            } else {
                denomination.value[0]
            }
        }
    }
```

다만 위와 같이 작성했을 경우 operation 로직이 너무 길어지기 때문에 별도 함수로 분리하는 걸 추천드려요!

Reply:
이런 방법이 있었군요!! 
감사합니다 :) 

두개의 커밋을 통해 수정했습니다.
**5ae24154**
**11b319f4**